syne_tune/optimizer/schedulers/searchers/searcher.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         self.config_space = config_space
         if points_to_evaluate is None:
             self.points_to_evaluate = []
+        else:
+            self.points_to_evaluate = points_to_evaluate
```

Comment:
Before we add new functionalities, can we test the ones we added in the previous PR? We should make sure that all our searchers and scheduler verify the same API (can suggest ~5 configurations, return suggestion which matches points to evaluate, returns suggestion that have the expected signature).

I believe that it would be better to do this before adding functionalities (such as multiobjective) as we want to make sure the functionality we work as expected.